packages/scheduler/src/Scheduler.js

```diff
@@ -261,21 +302,45 @@ function unstable_scheduleCallback(priorityLevel, callback, options) {
       typeof options.timeout === 'number'
         ? options.timeout
         : timeoutForPriorityLevel(priorityLevel);
+    // if (enableProfiling) {
+    //   var _label = options.label;
+    //   if (typeof _label === 'string') {
+    //     label = _label;
+    //   }
+    // }
```

Comment:
?

Reply:
That's from original PR. I can remove. I think it's a TODO.

Reply:
TODO itself:

https://github.com/facebook/react/blob/1f5f7df068a52aeb180797aa21dea1aef0329781/packages/scheduler/src/Scheduler.js#L292-L293

Reply:
I think it's safe to remove this TODO because I don't think we should put this feature in the profiler itself but instead expose the ID so that someone outside can associate the ID with a label if they want.